x/evm/statedb/statedb.go

```diff
@@ -0,0 +1,481 @@
+package statedb
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+type revision struct {
```

Comment:
comment: what it is and why is it used for

Reply:
done